lib/active_model/serializer/associations.rb

```diff
@@ -67,6 +65,15 @@ def has_one(name, options = {}, &block)
           associate(HasOneReflection.new(name, options, block))
         end
 
+        # Memoized default include tree (shared across all instances)
+        # @return [IncludeTree]
+        # @api private
+        def default_include_tree
+          @default_include_tree ||= ActiveModel::Serializer::IncludeTree.from_include_args(
+            ActiveModelSerializers.config.default_includes
+          )
+        end
```

Comment:
nice 'cause easy to override.  should this be a private api?  I'd consider making it public, but no need to delay this PR